### PR TITLE
Merge branch 'staging' of https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App into merge-staging-production

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/mobile-home/mobile-home-sidebar/mobile-home-sidebar.component.ts [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                    if (cmp && cmp.uniqueName) {
+                        return cmp.uniqueName === this.generalService.companyUniqueName;
+                    } else {
+                        return false;
+                    }
+                });
+
+                if (selectedCmp) {
+                    this.selectedCompany = cloneDeep(selectedCmp);
+                    console.log(this.selectedCompany);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/mobile-home/mobile-home-sidebar/mobile-home-sidebar.component.ts:55]: Remove console

File: apps/web-giddh/src/app/shared/header/header.component.ts [hunk 2]
```diff
@@ -1367,12 +1398,15 @@ export class HeaderComponent implements OnInit, AfterViewInit, OnDestroy, AfterV
 
     public openExpiredPlanModel(template: TemplateRef<any>) { // show expired plan
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/shared/header/header.component.ts:1399]: add return type

Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/header.component.ts:1399]: @rohit-walkover Did you check if this function is existing or new?

Comment (ravinderwalkover) [reply apps/web-giddh/src/app/shared/header/header.component.ts:1399]: @rohit-walkover did you check if this function is existing or new?

**Pull request merged**